src/index.tsx

```diff
@@ -1,17 +1,20 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
```

Comment:
Remove this file.

Reply:
Resolved on da0a0ae644a0fced279945ccd1a5797cd7b3fcd5